app/api/v2/models/user_model.py

```diff
@@ -0,0 +1,76 @@
+"""This file is the model that handle requests from views. Manipulates the database accordingly"""
+import psycopg2
+import jwt
+import json
+from datetime import datetime, timedelta
+from ....config import DataBase
+from ....config import app_config
+
+
+class UserModel():
+    """Handles the requests from exposed endpoints"""
+    def __init__(self):
+        self.db_connect = DataBase().db
+        self.migrations = DataBase().create_tables()
+    def add_user(self, data):
```

Comment:
Avoid obvious linting mistakes
```suggestion
    
    def add_user(self, data):
```

Reply:
Restructured various function defs to align with python standards